src/Directory.Build.props

```diff
@@ -35,6 +35,11 @@
     <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)CodeAnalysis.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(SolutionName)' == 'Sentry.Aot' and $(TargetFramework.StartsWith('net8'))">
+    <PublishAot>true</PublishAot>
+    <DefineConstants>$(DefineConstants);TRIMMABLE</DefineConstants>
```

Comment:
Is the target-framework net8 condition necessary here? Maybe we could instead define "Sentry.Aot" solution itself to target net8 (or net7 which, from my experience, seems to work fine too)?

Reply:
net9 comes out next year, and net10 after that. So checking versionOrLater is ideal